### ISSUE TITLE
Improvements on website

### ISSUE THREAD

**Issue by neo22s** [kohana/core#669]
From https://github.com/kohana/core/issues/660#issuecomment-192602891

I suggest we do a new home site, hosted at github pages with jekyll for example.

Something modern based on a template and yet simple no need to be complicated.

We need to point users to the right place clearly.

The home should not have more than:
- features
- showcase
- links to github, download, issues, kohana-modules.com, documentation, forum..

@sigmatec offered to help too

**Comment by sigma-technology** [comment]
I have a very good graphic designer in my employ along with a great CMS product built on Kohana. It sounds like a simple enough website, so I am happy to do this free of charge including hosting, so if this is of interest please let me know.

**Comment by neo22s** [comment]
Hello @sigmatec ,

Thanks for the offering the design I am sure is something we could use!

I do not think there's need of CMS or hosting we can use github pages with jekyll since its a static page ;)

lets see what others think about this.

**Comment by sigma-technology** [comment]
I'm not familiar with jekyll but as long as it allows us to have something hosted on the proper kohana domain then I'm in agreement.  My opinion is that there should be a website of some sort on kohanaframework.org to avoid it looking like an amateur product. I'm still happy for our team to do the graphic design, even if we don't build the website.

**Comment by enov** [comment]
@neo22s @sigmatec I thank you both for raising this issue and working on this, :100: 

I think the code of the website should be open sourced and should be available for Kohana maintainers to change in the future.

Another important thing is that the [documentations, including the API docs](https://kohanaframework.org/documentation), are also on the website. Each documentation version is in itself [a separate Kohana application](https://github.com/kohana/kohanaframework.org/tree/master/guides), a git submodule that is linked to a specific kohana/kohana repo commit.

Cheers!

**Comment by neo22s** [comment]
@enov I will leave the docs as they are just changing the colors/logo to the new site nothing else ;)

yes! code should be open sourced, so something like jekyll and hosted on GH makes lot of sense ;)

**Comment by enov** [comment]
@neo22s Github does not have a PHP runtime. Are we going to host the docs and the API refs somewhere else?

**Comment by neo22s** [comment]
@enov We can compile in local and upload statics ;) will need some tweaking but is doable. 

Generate a Markdown compatible or use a service suh as readthedocs or similar.

We use GH pages for https://docs.yclas.com/ using jekyll and a custom theme

**Comment by sigma-technology** [comment]
Would you like us to design a new logo for Kohana, to modernise it?  If so, how true to the current logo does it have to be?  I don't want to upset anyone who maybe designed the current one?

**Comment by neo22s** [comment]
@sigmatec I am not sure who makes this decision. 

What if we create a new issue with few logo suggestions and then vote?

Although is not something that worries me at all, I rather launch 4.0.0 asap ;)

**Comment by sigma-technology** [comment]
I agree 4.0.0 is more important :)  I can't contribute much to that though, so happy to do something to help promote Kohana to new users and expand the community.  I will get our Graphic Designer to come up with a couple of alternatives and then start an issue so that they can be discussed/voted on.
